frontend/src/components/Dashboard.vue

```diff
@@ -82,26 +89,57 @@
             :pieChartData="uniqueUserCountByRealm"
             v-if="!uniqueUserCountByRealmLoadingStatus"
           />
-            <v-skeleton-loader
+          <v-skeleton-loader
             v-if="uniqueUserCountByIDPLoadingStatus"
             ref="dashboardSkeleton"
-            width=''
-            type="image">
+            width=""
+            type="image"
+          >
           </v-skeleton-loader>
         </div>
       </div>
     </div>
+    <div class="flex">
+      <div class="tile" style="width: 1100px">
+        <div class="heading">
+          <p>Active Total User Count</p>
```

Comment:
I think "Active Users" is ambiguous so I'm trying to think of a better name. How about "User Login Events per Day"?

Also, could you experiment a little with reducing the number of X axis labels?

E.g. Before:

![image](https://user-images.githubusercontent.com/1767127/225805823-38281b97-4e4e-404b-bb99-fcf8c5853d2f.png)

After:
![image](https://user-images.githubusercontent.com/1767127/225805837-e2c15b92-49e8-4909-a480-c6b8180f07f9.png)

https://stackoverflow.com/a/68139050/201891